base/repl/REPLCompletions.jl

```diff
@@ -534,10 +536,11 @@ function completions(string, pos)
                         #   <Mod>/src/<Mod>.jl
                         #   <Mod>.jl/src/<Mod>.jl
                         if isfile(joinpath(dir, pname))
-                            endswith(pname, ".jl") && push!(suggestions, pname[1:end-3])
+                            endswith(pname, ".jl") && push!(suggestions,
+                                                            pname[1:prevind(pname, end-2)])
```

Comment:
This should have been OK, since we know the three last characters are ASCII. `-2` has no reason to be more correct than `-3`, right? Same below.

Reply:
OK, carry on, I need to learn how to count up to three. :-)